lib/integrations/externalcloudaudit/bootstrap_test.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package externalcloudaudit
+
+import (
+	"context"
+	"net/url"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/athena"
+	athenatypes "github.com/aws/aws-sdk-go-v2/service/athena/types"
+	"github.com/aws/aws-sdk-go-v2/service/glue"
+	gluetypes "github.com/aws/aws-sdk-go-v2/service/glue/types"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	s3types "github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/gravitational/trace"
+	"github.com/stretchr/testify/require"
+
+	ecatypes "github.com/gravitational/teleport/api/types/externalcloudaudit"
+)
+
+func TestBootstrapInfra(t *testing.T) {
+	tt := []struct {
+		desc        string
+		region      string
+		eca         *ecatypes.ExternalCloudAuditSpec
+		errExpected bool
+	}{
+		{
+			desc:        "nil input",
+			errExpected: true,
+		},
+		{
+			desc:        "standard input",
+			errExpected: false,
+			region:      "us-west-2",
+			eca: &ecatypes.ExternalCloudAuditSpec{
+				SessionsRecordingsURI:  "s3://long-term-storage-bucket/session",
+				AuditEventsLongTermURI: "s3://long-term-storage-bucket/events",
+				AthenaResultsURI:       "s3://transient-storage-bucket/query_results",
+				AthenaWorkgroup:        "teleport",
+				GlueDatabase:           "teleport",
```

Comment:
Could we use different strings, so we know the fields aren't being misused?

Reply:
Using different strings in [ffc9570](https://github.com/gravitational/teleport/pull/33272/commits/ffc95703e2d1522c1ebfb67a951be6683b284046).